package.json

```diff
@@ -1,17 +1,20 @@
 {
-  "name": "folder-hash",
-  "version": "4.0.4",
+  "name": "@aeolun/folder-hash",
```

Comment:
Can you keep this rename in your fork ;) ?  

I would still like to add the first commit and maybe you can create a PR from another branch than the main one?
Then you can keep working on your fork without updating this PR.